src/api/routes/guilds/templates/index.ts

```diff
@@ -140,25 +140,25 @@ router.post(
 
 		const guild_id = Snowflake.generate();
 
-		const [guild] = await Promise.all([
-			Guild.create({
-				...body,
-				...template.serialized_source_guild,
-				id: guild_id,
-				owner_id: req.user_id,
-			}).save(),
-			Role.create({
-				id: guild_id,
-				guild_id: guild_id,
-				color: 0,
-				hoist: false,
-				managed: true,
-				mentionable: true,
-				name: "@everyone",
-				permissions: BigInt("2251804225").toString(), // TODO: where did this come from?
-				position: 0,
-			}).save(),
-		]);
+		const guild = await Guild.create({
+			...body,
+			...template.serialized_source_guild,
+			id: guild_id,
+			owner_id: req.user_id,
+			premium_tier: 0,
```

Comment:
Here, I'm setting `premium_tier` to 0. This is an example of Option 2 from my list in the PR.